### DISCUSSION TITLE
Unreliable routers in the Zigbee mesh

### DISCUSSION THREAD

**Discussion by sohnvonoff** [Koenkk/zigbee2mqtt#8353] [General]
I have a few Zigbee lamps in my Zigbee network. These lamps are often disconnected from the electricity. As a result, other devices are temporarily no longer accessible to me because the lights that are switched off also fail as routers. It takes some time for the Zigbee mesh to work again. Sometimes the node that can no longer be reached has to be reset. 
My question: Can you set / configure a router as an end node? 

Thanks in advance.

My System: Pi4 w. Hassio, zigbee2mqtt w. Conbee2

**Comment by DubhAd** [comment]
No, that's handled entirely by the device itself.

The answer is to either buy lights that are end devices (some behave that way apparently), to not turn off the lights, or to replace them with dumb bulbs and make the control for them smart.
